index.html

```diff
@@ -1420,16 +1422,20 @@ <h3>Multi-Cloud</h3>
     </section>
 
     <section id="virtual-thing">
-      <h3>Virtual Things</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a <a>virtual thing</a> scenario,
-      as described in the use case for 
-      <a href="https://w3c.github.io/wot-usecases/#USE-CASES/Retail-virtual-thing">Virtual Thing</a>
+      <h3>Virtual Things</h3>   
+	<p>
+	 A virtual thing is an abstraction of a physical entity that does not have a direct physical counterpart.
+	 It can be used to describe abstract interfaces with a TD that are decoupled from individual devices. 
```

Comment:
Until this line I was OK but then after this line, it sounds too much like TM. It is also a bit wrong to mix the two.

Reply:
@egekorkan 
Please be more specific about your concerns. Please review the virtual thing use case in the use case document.

Reply:
I agree, "abstract interfaces" sounds too much like what a TM is trying to do.  It may be explained elsewhere, but that does not mean this statement isn't confusing.  If you remove the word "abstract" it might be ok.  For example, it could be a software-only service or simulation, which has an interface but is not a device.  To me "abstract" means "without an instantiation" which describes a Thing Model, not a running simulation or service (which is instantiated.  It is virtual, but not abstract).

Reply:
> @egekorkan Please be more specific about your concerns. Please review the virtual thing use case in the use case document.

I had a look and the main issue seems to be the use cases document. Should I open an issue there?